flipchem/msis.py

```diff
@@ -59,11 +62,12 @@ def compute_ion_neutral_collfreq(densities, Tn, mi, Ti=None):
     #    n_in = const * n_n / sqrt(m_i*(m_i+m_n))
     # for each ion. Here's the constants for H+, He+, N+, O+, N2+, and O2+:
     Hconst  = 2.118436361550408e-15
-    Heconst = 2.372016271579909e-15 # assuming 100% He-4
+    Heconst = 4.796838917e-15 #old value: 2.372016271579909e-15 # assuming 100% He-4
```

Comment:
```suggestion
    Heconst = 4.796838917e-15 # From 10.1103/PhysRevLett.120.123203
```

Reply:
We'll have the old value in the git history, so we can remove that, but it's probably useful to start leaving comments about where these constants are coming from now.